src/components/taskDetails/Details.tsx

```diff
@@ -1,20 +1,110 @@
-import React, { FC, useState } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import moment from 'moment';
 import { FaReceipt } from 'react-icons/fa6';
 import Link from 'next/link';
 import Tooltip from '@/components/common/Tooltip/Tooltip';
 import setColor from './taskPriorityColors';
 import extractRepoName from '@/utils/extractRepoName';
 import styles from './task-details.module.scss';
-import { TaskDetailsProps } from '@/interfaces/taskDetails.type';
+import {
+    DetailsContentProps,
+    TaskDetailsProps,
+} from '@/interfaces/taskDetails.type';
+import useUserData from '@/hooks/useUserData';
 import { STARTED_ON, ENDS_ON } from '@/constants/constants';
+import { isValidDate } from '@/utils/isValidDate';
 
 type StringNumberOrUndefined = string | number | undefined;
 
-const Details: FC<TaskDetailsProps> = ({ detailType, value, url }) => {
+const DetailsContent: FC<DetailsContentProps> = ({
+    color,
+    isGitHubLink,
+    value,
+    gitHubIssueLink,
+    isTimeDetail,
+    formatDate,
+    tooltipActive,
+    renderedValue,
+    getRelativeTime,
+}) => {
+    const displayValue = renderedValue || value;
+
+    if (isGitHubLink && value && gitHubIssueLink) {
+        return (
+            <span
+                className={styles.detailValue}
+                style={{ color: color ?? 'black' }}
+            >
+                <a
+                    className={styles.gitLink}
+                    href={gitHubIssueLink}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    aria-label="Open GitHub Issue"
+                    title={value}
+                >
+                    {extractRepoName(value)}
+                </a>
+            </span>
+        );
+    }
+
+    if (isTimeDetail && value) {
+        return (
+            <span
+                className={styles.detailValue}
+                style={{ color: color ?? 'black' }}
+            >
+                <Tooltip
+                    content={formatDate(value)}
+                    tooltipPosition={{
+                        top: '-3.6rem',
+                        right: '-4.5rem',
```

Comment:
Do we need negative value here?

Reply:
It was previously there and for tooltip position alignment it is required.